### PR TITLE
fix: [CDS-40879]: panel for environemnt fix

### PR DESCRIPTION
**Pull request by dhruvUpadhyay07**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by YounixIO — APPROVED** [review]

**Review by kapilkumawat86 — DISMISSED** [review]

File: src/modules/70-pipeline/components/PipelineStudio/StageBuilder/StageBuilder.tsx [hunk 1]
```diff
@@ -783,7 +783,7 @@ function StageBuilder(): JSX.Element {
     switch (sectionIdPassed) {
       case 'SERVICE':
         return 'ServicePipelineStudio'
-      case 'INFRASTRUCTURE':
```
Comment (AshwinBhatkal-Harness) [inline src/modules/70-pipeline/components/PipelineStudio/StageBuilder/StageBuilder.tsx:786]: Are we sure on this? If we move this to ENVIRONMENT, we lose context to INFRASTRUCTURE where feature flag is not enabled?

**Comment by dhruvUpadhyay07** [comment]
retrigger titlecheck

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Review by AshwinBhatkal-Harness — DISMISSED** [review]

**Review by kapilkumawat86 — DISMISSED** [review]

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/10226)

**Comment by dhruvUpadhyay07** [comment]
retrigger jest

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**